src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -6,6 +6,7 @@
 
 from elasticsearch_dsl import A, Q, Search
 
+from azul import config as azul_config
```

Comment:
The convention is that this is imported under the original name `config`.

To avoid the conflict with service.config could you import that as service_config?